content/docs/08-user-management/4-saml-sso.md

```diff
@@ -39,35 +39,40 @@ The two types of SSO authentication methods Palette supports are: <p></p><br />
 
 # How to Set Up SAML 2.0-Based SSO within Palette
 
-With Palette, you can use SAML 2.0 single sign-on (SSO) to enable authentication using an IdP when logging into Palette.   
- 
-## Procedure 
+With Palette, you can use SAML 2.0 single sign-on (SSO) to enable authentication using an IdP when logging into Palette.
+
+## Procedure
 
 1. To set up an Identity provider (IdP) based SSO, log in to the Palette console as the Tenant Admin.<p></p><br />
 2. Select the **Tenant Settings** > **SSO Auth Type** > **SAML** to view the SAML panel.<p></p><br />
 3. Complete the assertion form with the requested parameters. See below for more details specific to supported IdPs.<p></p><br />
- 
-     - Issuer - The unique identifier. In this case it is Palette.
-     - Certificate - This certificate is generated (from where?)
-     - Service - 
+
+     - Issuer - The unique identifier. In this case, it is Palette.
+     - Certificate
+     - Service
      - Identity Provider Metadata
-     - Default Teams -
-         - Enable Single Logout -
- 
-4. These parameters enable Palette as a **Service Provider** (SP) into the IdP:<br /> (Pre-configured from the …)
-     - Single Logout URL -
-     - EntityId - 
-     - NameId Format - 
-     - Login URL - 
-     - FirstName -
-     - LastName -
-     - Email - 
-     - SpectroTeam - 
-     - Service Provider Metadata -
-    
-4. Copy the **Identity Provider Metadata** from the IdP into the Palette SAML panel for each metadata. {There is no Metadata? Do we need this step?}
- 
-5. Click **Enable** to complete the setup wizard.
+     - Default Teams
+         - Enable Single Logout
+
+<p></p>
+
+4. These parameters enable Palette as a **Service Provider** (SP) into the IdP:<br />
+   - Single Logout URL
+   - EntityId
+   - NameId Format
+   - Login URL
+   - FirstName
+   - LastName
+   - Email
+   - SpectroTeam
+   - Service Provider Metadata
+
+<p></p>
+
+5. Copy the **Identity Provider Metadata** from the IdP into the Palette SAML panel for each metadata. {There is no Metadata? Do we need this step?}
```

Comment:
Yes, and good question. Tis is needed if you use SAML, and left blank if you use OIDC.